### ISSUE TITLE
Features need documenting

### ISSUE THREAD

**Issue by AdamHess** [Sitecore-Community/Sitecore-Mvc-Contrib#20]
While I understand the goal of the project is to make our lives easier working with sitecore, this project could benefit from better documentation of existing features and how to use them.

**Comment by kevinobee** [comment]
@AdamHess you are absolutely right. The project is out of date and could definitely do with some TLC.

I've passed this feedback to @herskinduk and I would hope to see some updates around the Sitecore 8.2 release timeframe.
